### ISSUE TITLE
Nuxt3 generated Sites do not pass W3C validator

### ISSUE THREAD

**Issue by volkerjaenisch** [nuxt/framework#4391]
### Environment

uxt project info:                                                                                                                                                                  03:13:13

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.1`
- Nuxt Version:     `3.0.0-27500677.83d27bd`
- Package Manager:  `npm@8.5.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

Just call:

https://validator.w3.org/nu/?doc=https%3A%2F%2Fneu.datenadler.de

### Describe the bug

We checked twice that these errors do no originate from our code. They are coming from the framework.

```
Error: Bad value module for attribute type on element [link](https://html.spec.whatwg.org/multipage/#the-link-element): Subtype missing. From line 5, column 1534; to line 5, column 1609
5db.woff"><link type="module" rel="prefetch" href="/_nuxt/accessibility-a5c1e843.mjs"><link
```

```
Error: A script element with a defer attribute must not have a type attribute with the value module.
From line 9, column 14677; to line 9, column 14736
)</script><script type="module" src="/_nuxt/entry-fe88f193.mjs" defer></scri
```

**Comment by danielroe** [comment]
If you run `yarn upgrade nuxt3` or `npm up nuxt3`, this should now be resolved in v0.3.6 of `vue-bundle-renderer`.

**Comment by volkerjaenisch** [comment]
Dear @danielroe !
Thank you so much for this fine and fast solution. Works like a charm.
Cheers,
Volker
